privateGPT.py

```diff
@@ -17,12 +18,13 @@
 
 from constants import CHROMA_SETTINGS
 
-def main():
+def main(hide_source=False, mute_stream=False):
```

Comment:
As per the other comment, this should remain just `def main():`
Also note that `hide_source` and `mute_stream` are already set to `False` by default because of `action='store_true'`

Reply:
Yep, moved it inside the main as per the previous comment and remove this, keeping ```def main():``` as is